.github/workflows/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,59 @@
+name: 🚀 Feature request
+description: Add contributors to guestbook 💡
+title: "Feature:"
+labels: [💡 feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+  - type: dropdown
+    attributes:
+      multiple: false
+      label: Type of feature
+      description: Select the type of feature request, the lowercase should also be the PR prefix.
+      options:
+        - "🍕 Feature"
+        - "🐛 Fix"
+        - "📝 Documentation"
+        - "🎨 Style"
+        - "🧑‍💻 Refactor"
+        - "🔥 Performance"
+        - "✅ Test"
+        - "🤖 Build"
+        - "🔁 CI"
+        - "📦 Chore"
+        - "⏩ Revert"
+    validations:
+      required: true
```

Comment:
This part is not mentioned in the issue.

Reply:
Still trying to figure out what to remove

Reply:
The whole type. :)

You can test by creating a repo and copy paste what you have here in that repo.
Then further test it by creating an issue.

Reply:
Ooh

Reply:
> Still trying to figure out what to remove

It'll be easier for you to fix format or remove parts when you can see the form live.

Thanks for removing! 👍